BenchmarkSample/ui/src/main/java/com/example/benchmark/ui/MainActivity.kt

```diff
@@ -1,17 +1,16 @@
 package com.example.benchmark.ui
 
 import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
-import androidx.recyclerview.widget.DiffUtil
-import androidx.recyclerview.widget.ListAdapter
+import androidx.lifecycle.ViewModel
 import androidx.recyclerview.widget.RecyclerView
 
 class MainActivity : AppCompatActivity() {
+
+    // It's used in benchmarks
+    @Suppress("MemberVisibilityCanBePrivate")
```

Comment:
@TestOnly?

Reply:
hm, is it TestOnly? It's called from the class itself as well.